app/src/main/java/org/mozilla/fenix/perf/Stat.kt

```diff
@@ -25,6 +25,9 @@ private const val FIELD_POS_STARTTIME = 21 // starttime naming matches field in
 open class Stat {
 
     @VisibleForTesting(otherwise = PRIVATE)
+    /**
+     * @throws [java.io.FileNotFoundException]
+     */
```

Comment:
nit: I'd add a kdoc to `getFrameworkStartNanos` too. It's more important than this method because it's the non-private method of this class